packages/base/src/config/NoConflict.js

```diff
@@ -1,9 +1,37 @@
 import { getNoConflict as getConfiguredNoConflict } from "../InitialConfiguration.js";
 
+// Fire these events even with noConflict: true
+const excludeList = [
+	"value-changed",
+];
+
+const shouldFireOriginalEvent = eventName => {
+	return excludeList.includes(eventName);
+};
+
 let noConflict = getConfiguredNoConflict();
 
-const getNoConflict = () => {
-	return noConflict;
+const shouldNotFireOriginalEvent = eventName => {
+	return !(noConflict.events && noConflict.events.includes && noConflict.events.includes(eventName));
+};
+
+const getNoConflict = eventName => {
```

Comment:
Leave getNoConflict unchanged, export skipOriginalEvent and use it in UI5Element